pkg/tar/chroot.go

```diff
@@ -0,0 +1,113 @@
+// Copyright 2015 The rkt Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package tar
+
+import (
+	"archive/tar"
+	"encoding/json"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strconv"
+	"syscall"
+
+	"github.com/coreos/rkt/pkg/multicall"
+	"github.com/coreos/rkt/pkg/sys"
+)
+
+const (
+	multicallName = "extracttar"
+	fileMapFdNum  = 3
+)
+
+var mcEntrypoint multicall.Entrypoint
+
+func init() {
+	mcEntrypoint = multicall.Add(multicallName, extractTarCommand)
+}
+
+func extractTarCommand() error {
+	if len(os.Args) != 3 {
+		return fmt.Errorf("not enough arguments")
```

Comment:
Right! Is it good enough `Usage: extracttar DIR {true|false}`? As for simplicty I avoided the use of flags I'm not sure which is the best way to explain that the second parameter is for enabling/disabling files overwrite.